### ISSUE TITLE
A PHP ERROR 

### ISSUE THREAD

**Issue by ghost** [proimage/MC_Player#1]
DEAR MICHAEL   

THANK YOU FOR THIS GREAT IMPLEMENTATION OF JWP 

 I HAVE ENCOUNTERED AN ERROR IN THE MC_PLAYER 
THIS HAPPENS ON FIREFOX  IN THE PLUGINS HEADER PAGE AND ON THE TEMPLATE PAGE WHERE I PLACE THE TAG        {exp:mc_player:play file="/JWP/video.mp4" width="320" height="240" playerpath="/JWP/player.swf"}
 ON SAFARI SURPRISINGLY IT WORKS FINE 
IT GOES LIKE THIS :
Severity: Warning

Message: Cannot modify header information - headers already sent by (output started at /localhost/system/expressionengine/third_party/mc_player/pi.mc_player.php:1)

Filename: core/Common.php

Line Number: 409

PLEASE ALSO NOTE THAT I AM ON THE LATEST VERSION 0.2.1 (2011-11-22)

YOUR ADVISE IS MUCH APPRECIATED 

THANKS AGAIN ,

KALQAATRI

KALQAATRI@GMAIL.COM

**Comment by proimage** [comment]
Hi Kalqaatri,
I'm unable to reproduce the problem you're getting, but I've encountered this type of thing before. Sometimes, when a file is encoded in UTF-8, there is an invisible "header", called a Byte-Order Mark, or BOM, that is at the very front of the file - even plain "text" files such as .php files. When these kinds of files are processed by the server, the BOM can sometimes be seen as output, thus leading to the error message you encountered.

So I can't say for sure, but give 0.2.2 a try - I changed the encoding from "UTF-8" to "UTF-8 without BOM" (as shown in Notepad++), so that may fix it; let me know.
